bindings/jupyter/mlvis/widget_builder.py

```diff
@@ -1,71 +1,30 @@
 # Dynamically build component wrappers utilizing the ipywidget
 import os, sys, json, re, importlib
+from pkg_resources import resource_string
+from ast import literal_eval
 from traitlets import Unicode
 from .widget import CommonComponent
-from .widget_ext import Manifold, GraphBuilder, StackedCalendar, FeatureListView, MultiWayPlot
+from .widget_ext import Manifold
 
 
-components = [
-  'FeatureListView',
-  'GraphBuilder',
-  'Manifold',
-  'MultiWayPlot',
-  'StackedCalendar'
-]
+extensions = {'Manifold': Manifold}
```

Comment:
why do you remove all other packages?

Reply:
Now the components are loaded according to the jrequirements.json specification:
```python
def load_jrequirements():
    return literal_eval(resource_string('mlvis', 'jrequirements.json').decode('utf8'))

# Extract the Python component wrapper names from the jrequirements
def extract_components(jrequirements):
    return [component for component in jrequirements]

jrequirements = load_jrequirements()
components = extract_components(jrequirements)
```